garrysmod/lua/vgui/dscrollpanel.lua

```diff
@@ -27,119 +30,106 @@ function PANEL:Init()
 	self:SetPaintBackgroundEnabled( false )
 	self:SetPaintBorderEnabled( false )
 	self:SetPaintBackground( false )
-
 end
 
 function PANEL:AddItem( pnl )
-

```

Comment:
Pointless whitespace changes (Many other instances not commented on)

Reply:
You will have to follow code formatting style of the repository if you wish your code to be merged. Simple as that.

The code formatting style is chosen in such a way to be easier to read.

Reply:
No, I understand when the style has to match the repository style, but wouldn't it be easier to just remove all those spaces, it's a bunch of extra characters that take up space and make no sense, why don't you just remove them all from the repository, they are not part of the code..